diff --git a/chrome/browser/ui/libgtkui/gtk_util.cc b/chrome/browser/ui/libgtkui/gtk_util.cc
index bb6102ef0cfa19f95ccf94e680ab8806ea473635..21cdcc5efb2c2a3bb1aa204878a1f87be2fe130a 100644
--- a/chrome/browser/ui/libgtkui/gtk_util.cc
+++ b/chrome/browser/ui/libgtkui/gtk_util.cc
@@ -85,7 +85,7 @@ std::string GetDesktopName(base::Environment* env) {
   std::string name;
   if (env->GetVar("CHROME_DESKTOP", &name) && !name.empty())
     return name;
-  return "chromium-browser.desktop";
+  return "dissenter-browser.desktop";
 #endif
 }
 
