diff --git a/chrome/installer/linux/common/desktop.template b/chrome/installer/linux/common/desktop.template
index 62f031cfdff3ae9a7b25b76f88b3ba40ab477e75..289e3d0346e1649a723aa7baa8811809ec34e169 100644
--- a/chrome/installer/linux/common/desktop.template
+++ b/chrome/installer/linux/common/desktop.template
@@ -105,7 +105,7 @@ Comment[vi]=Truy cập Internet
 Comment[zh_CN]=访问互联网
 Comment[zh_HK]=連線到網際網路
 Comment[zh_TW]=連線到網際網路
-Exec=/usr/bin/@@USR_BIN_SYMLINK_NAME@@ %U
+Exec=/usr/bin/@@USR_BIN_SYMLINK_NAME@@ %U --class=dissenter-browser
 StartupNotify=true
 Terminal=false
 Icon=@@PACKAGE@@
@@ -166,7 +166,7 @@ Name[uk]=Нове вікно
 Name[vi]=Cửa sổ Mới
 Name[zh_CN]=新建窗口
 Name[zh_TW]=開新視窗
-Exec=/usr/bin/@@USR_BIN_SYMLINK_NAME@@
+Exec=/usr/bin/@@USR_BIN_SYMLINK_NAME@@ --class=dissenter-browser
 
 [Desktop Action new-private-window]
 Name=New Incognito Window
@@ -218,4 +218,4 @@ Name[uk]=Нове вікно в режимі анонімного перегля
 Name[vi]=Cửa sổ ẩn danh mới
 Name[zh_CN]=新建隐身窗口
 Name[zh_TW]=新增無痕式視窗
-Exec=/usr/bin/@@USR_BIN_SYMLINK_NAME@@ --incognito
+Exec=/usr/bin/@@USR_BIN_SYMLINK_NAME@@ --incognito  --class=dissenter-browser
