diff --git a/chrome/browser/resources/settings/about_page/about_page.js b/chrome/browser/resources/settings/about_page/about_page.js
index 7f9047f7226b76c675f649cd7cc0b94124b591b5..23c5b854e802c04eff9c36a919d4f0b0d96e99df 100644
--- a/chrome/browser/resources/settings/about_page/about_page.js
+++ b/chrome/browser/resources/settings/about_page/about_page.js
@@ -207,12 +207,6 @@ Polymer({
     });
 
     // </if>
-    // <if expr="not chromeos">
-    this.startListening_();
-    // </if>
-    if (settings.getQueryParameters().get('checkForUpdate') == 'true') {
-      this.onCheckUpdatesTap_();
-    }
   },
 
   /**
@@ -231,6 +225,7 @@ Polymer({
 
   /** @private */
   startListening_: function() {
+    /*
     this.addWebUIListener(
         'update-status-changed', this.onUpdateStatusChanged_.bind(this));
     // <if expr="_google_chrome and is_macosx">
@@ -245,6 +240,7 @@ Polymer({
         this.onTPMFirmwareUpdateStatusChanged_.bind(this));
     this.aboutBrowserProxy_.refreshTPMFirmwareUpdateStatus();
     // </if>
+    */
   },
 
   /**
@@ -303,7 +299,14 @@ Polymer({
 
   /** @private */
   onRelaunchTap_: function() {
+    // <if expr="is_macosx">
+    // Sparkle framework's relaunch api is used.
+    this.lifetimeBrowserProxy_.relaunchOnMac();
+    // </if>
+
+    // <if expr="not is_macosx">
     this.lifetimeBrowserProxy_.relaunch();
+    // </if>
   },
 
   /** @private */
