diff --git a/components/os_crypt/key_storage_linux.cc b/components/os_crypt/key_storage_linux.cc
index 9e6f0f27535fdf52a23b5a734c108196ae0b728c..a8206eebd57c40f0e63ae65a1ec969199b096712 100644
--- a/components/os_crypt/key_storage_linux.cc
+++ b/components/os_crypt/key_storage_linux.cc
@@ -29,8 +29,8 @@
 const char KeyStorageLinux::kFolderName[] = "Chrome Keys";
 const char KeyStorageLinux::kKey[] = "Chrome Safe Storage";
 #else
-const char KeyStorageLinux::kFolderName[] = "Chromium Keys";
-const char KeyStorageLinux::kKey[] = "Chromium Safe Storage";
+const char KeyStorageLinux::kFolderName[] = "Dissenter Keys";
+const char KeyStorageLinux::kKey[] = "Dissenter Safe Storage";
 #endif
 
 // static
