diff --git a/components/update_client/updater_state_win.cc b/components/update_client/updater_state_win.cc
index 89c2c186f9c095d6c2a9210ca2c4e8a94ee64b98..9ba9fb9cb2c32bc3c03f22b3643ffdd0de76d25e 100644
--- a/components/update_client/updater_state_win.cc
+++ b/components/update_client/updater_state_win.cc
@@ -25,20 +25,35 @@ namespace {
 
 // Google Update group policy settings.
 const wchar_t kGoogleUpdatePoliciesKey[] =
+#if defined(BRAVE_CHROMIUM_BUILD)
+    L"SOFTWARE\\Policies\\GabAI\\Update";
+#else
     L"SOFTWARE\\Policies\\Google\\Update";
+#endif
 const wchar_t kCheckPeriodOverrideMinutes[] = L"AutoUpdateCheckPeriodMinutes";
 const wchar_t kUpdatePolicyValue[] = L"UpdateDefault";
 const wchar_t kChromeUpdatePolicyOverride[] =
+#if defined(BRAVE_CHROMIUM_BUILD)
+    L"Update{0C12B489-0B55-4F8B-8EB7-640256ADBE54}";
+#else
     L"Update{8A69D345-D564-463C-AFF1-A69D9E530F96}";
+#endif
 
 // Don't allow update periods longer than six weeks (Chrome release cadence).
 const int kCheckPeriodOverrideMinutesMax = 60 * 24 * 7 * 6;
 
 // Google Update registry settings.
+#if defined(BRAVE_CHROMIUM_BUILD)
+const wchar_t kRegPathGoogleUpdate[] = L"Software\\GabAI\\Update";
+const wchar_t kRegPathClientsGoogleUpdate[] =
+    L"Software\\GabAI\\Update\\Clients\\"
+    L"{B131C935-9BE6-41DA-9599-1F776BEB8019}";
+#else
 const wchar_t kRegPathGoogleUpdate[] = L"Software\\Google\\Update";
 const wchar_t kRegPathClientsGoogleUpdate[] =
     L"Software\\Google\\Update\\Clients\\"
     L"{430FD4D0-B729-4F61-AA34-91526481799D}";
+#endif
 const wchar_t kRegValueGoogleUpdatePv[] = L"pv";
 const wchar_t kRegValueLastStartedAU[] = L"LastStartedAU";
 const wchar_t kRegValueLastChecked[] = L"LastChecked";
