diff --git a/components/os_crypt/key_storage_linux.cc b/components/os_crypt/key_storage_linux.cc
index 33fed0fa438776e3da4de6a3589745ad8b8304e9..3282798893c8d74ca7954c64b56b35e49f2480d4 100644
--- a/components/os_crypt/key_storage_linux.cc
+++ b/components/os_crypt/key_storage_linux.cc
@@ -30,8 +30,8 @@
 const char KeyStorageLinux::kFolderName[] = "Chrome Keys";
 const char KeyStorageLinux::kKey[] = "Chrome Safe Storage";
 #else
-const char KeyStorageLinux::kFolderName[] = "Chromium Keys";
-const char KeyStorageLinux::kKey[] = "Chromium Safe Storage";
+const char KeyStorageLinux::kFolderName[] = "Dissenter Keys";
+const char KeyStorageLinux::kKey[] = "Dissenter Safe Storage";
 #endif
 
 namespace {
